README.md

```diff
@@ -47,10 +47,10 @@ Then run
 :PluginInstall
 ```
 
-If you are using Vim 8+, you don't need any plugin manager. Simply clone this repository inside `~/.vim/pack/plugin/start/` directory and restart Vim.
+If you are using Vim 8+, you don't need any plugin manager. Simply clone this repository inside `~/.vim/pack/plugin/start/config/` directory and restart Vim.
```

Comment:
Hmm, I'll be honest that I'm still using vim-plug so I'm less well versed with the new native package manage stuff, but this doesn't look quite right to me. Based on [these docs](https://vimhelp.org/repeat.txt.html#package-create), it looks like perhaps we're missing a top-level `vim/pack/start/vim-tmux-navigator` directory or similar. Mind taking a look at that?

Reply:
See https://vi.stackexchange.com/questions/613/how-do-i-install-a-plugin-in-vim-vi

It seems like we should actually be doing s/plugin/bundle

This change worked for me. See also my comment below.